src/eos.py

```diff
@@ -63,26 +79,41 @@ def header(self):
 class LocallyIsothermalEOS(EOS_Table):
     '''Simple locally isothermal power law equation of state:
 
-    args:
-        cs0     : sound-speed at 1AU
-        q       : power-law index of sound-speed
-        alpha_t : turbulent alpha parameter
-        star    : stellar properties
-        mu      : mean molecular weight, default=2.4
-    '''
-    def __init__(self, star, cs0, q, alpha_t, mu=2.4):
-        super(LocallyIsothermalEOS, self).__init__()
+    	args:
+    		star    : stellar properties
+        	cs0     : sound speed at 1 AU
+        	beta 	: power-law index of the aspect ratio H/R as a function of radius
```

Comment:
Are we sure beta is the index as a function of radius? That just depends on q, doesn't it? I think beta is as a function of stellar mass (also looking at the code below...)?